bin/inference/pycbc_inference

```diff
@@ -128,29 +134,9 @@ with ctx:
     # Note: the pool is created at this point. This means that,
     # unless you enjoy angering your cluster admins,
     # NO SAMPLES FILE IO SHOULD BE DONE PRIOR TO THIS POINT!!!
-    sampler = inference.sampler.load_from_config(
-        cp, model, nprocesses=opts.nprocesses, use_mpi=opts.use_mpi)
-
-    # set up output/checkpoint file
-    sampler.setup_output(opts.output_file, force=opts.force)
-
-    # Figure out where to get the initial conditions from: a samples file,
-    # the checkpoint file, the prior, or an initial prior.
-    samples_file = opts.samples_file
-    # use the checkpoint file instead if resume from checkpoint
-    if not sampler.new_checkpoint:
-        samples_file = sampler.checkpoint_file
-    if samples_file is not None:
-        logging.info("Initial positions taken from last iteration in %s",
-                     samples_file)
-        init_prior = None
-    else:
-        # try to load an initial distribution from the config file
-        init_prior = inference.sampler.initial_dist_from_config(
-            cp, sampler.variable_params)
-
-    sampler.set_initial_conditions(initial_distribution=init_prior,
-                                   samples_file=samples_file)
+    sampler = inference.sampler.load_from_config(cp, model,
+        output_file=opts.output_file, nprocesses=opts.nprocesses,
+        use_mpi=opts.use_mpi)
```

Comment:
Still PEP8 issue... put `cp, model` on new line... it's fine to just indent on a line continuation, but if you do, you're not supposed to have anything after the `(` on the first line. Otherwise, if you put things on the same line as the `(` you have to align all subsequent things with that.